packages/fx-core/src/plugins/resource/aad/constants.ts

```diff
@@ -268,3 +268,10 @@ export class TemplatePathInfo {
     "bicep"
   );
 }
+
+export class ConfigFilePath {
+  static readonly Default = "env.default.json";
+  static readonly LocalSettings = "localSettings.json";
+  static readonly Profile = (env: string) => `profile.${env}.json`;
+  static readonly Input = (env: string) => `azure.parameters.${env}.json`;
```

Comment:
The existing AAD configuration should be set in `config.${env}.json` rather than `azure.parameters.${env}.json`.
No matter user choose to use an existing AAD or create a new one, the AAD info will be persistent in the output file - `profile.${env}.json`. And the value of the  AAD client id and secret in `azure.parameters.${env}.json` will be resolved from `profile.${env}.json`

Reply:
Will update, thx!

Reply:
Fixed, thx!